packages/table/src/docs.md

```diff
@@ -1,13 +1,12 @@
+---
```

Comment:
Are we putting the copyright header in the .md files? I omitted it from `core` and `datetime`